src/main/java/com/google/sps/util/TestUtil.java

```diff
@@ -50,8 +51,8 @@ public static Entity createRecipeEntity(String entityName, String name, String a
     for (int i = 0; i < numRecipes; i++) {
       testRecipes[i] = TestUtil.createRecipeEntity(
         entityName, recipeName, author,
-        message + String.format(" I AM RECIPE %d OF MY KIND.", i + offset),
-        tags, ingredients, steps, Integer.valueOf(0));
+        message + String.format(". I AM RECIPE %d OF MY KIND.", i + offset),
```

Comment:
What is this additional period at the beginning? Slash what is `message` here?

Reply:
The message is meant to be a string saying what test the recipe is from. For example, in the test files I currently have, it says "THIS IS A TEST RECIPE FOR THE SEARCH BY NAME TEST." This goes in the recipe description box and makes it easy to see whether or not the correct recipe data is being used in a search test. The period helps with formatting, since the string from where it's called had no punctuation. But I think I will move it to the actual method, so that formatting in the future makes more sense.